src/ExpenseTracker/Models/Account.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace ExpenseTracker.Models;
+
+/// <summary>
+/// Represents the account with name, balance, list of transactions.
+/// </summary>
+public class Account : IAccount
+{
+    /// <summary>
+    /// Gets current balance of the account.
+    /// </summary>
+    /// <value>
+    /// Current balance of the account.
+    /// </value>
```

Comment:
```suggestion
    /// <inheritdoc/>
```

Update all the necessary places